src/steps/command/plugins.ts

```diff
@@ -12,6 +12,8 @@ export class Plugin {
 
 export interface Plugins<T> {
     add(plugin: Plugin): T;
+    /** Get the first plugin that matches the given predicate */
+    get(predicate: (plugin: Plugin) => boolean): Plugin | undefined;
```

Comment:
this is not complete, as it only ever allows you to get the first one - can we just expose the `plugins` array as readonly?

Reply:
I could change it to return all matching plugins instead.

I put this in the PR about adding `readonly plugins: Plugin[]` instead:
> One other option was to add a public `plugins: Plugin[]` array to the interface, but that would then enable all array methods (do we want this? .push() would conflict with the current add() method).

Thoughts on that? There would be little need to keep the `add()` method since you could chain adding plugins with `.push(...plugins)`, though I don't mind keeping `add()` anyway.

Reply:
Exposing `readonly plugins: Plugin[]` would also mean you use it like this:
```ts
step.plugins.plugins.find(<something>);
```

Reply:
Opened #47 as well / instead to add `readonly plugins: Plugin[]` to the interface instead. Happy for either to be merged in.